src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1408,6 +1410,18 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     self->flags                            = 0;
     self->tx.av_fl_mlid                    = self->super.super.super.path_bits[0] & 0x7f;
 
+    if (config->fc_hard_req_timeout == UCS_ULUNITS_AUTO) {
```

Comment:
why do we need to support auto value? Just default value would be enough.
You can define the var as `UCS_CONFIG_TYPE_TIME` to prohibit `auto`

Reply:
>why do we need to support auto value? Just default value would be enough.

but default is `5s`, so we should handle `auto` value to convert to `5s`.

>You can define the var as UCS_CONFIG_TYPE_TIME to prohibit auto

then we won't have a possibility to set `inf` value which is useful sometimes.

Reply:
ok, then can just print error if `auto` is specified.

Reply:
ok, done